internal/announce/announce.go

```diff
@@ -199,6 +199,9 @@ func (a *announceProcessor) onLinesMatched(def *domain.IndexerDefinition, vars m
 		return err
 	}
 
+	// replace all – with - in torrentName var (needed for proper parsing with rls)
+	rls.TorrentName = strings.ReplaceAll(rls.TorrentName, "–", "-")
```

Comment:
This feels like opening a can of worms with unexpected behavior.

Might be better to use the Sprig Replace function on `torrentName` for the indexer definition instead.

http://masterminds.github.io/sprig/strings.html#replace

Reply:
If you have a solution in your head, i would appreciate you implementing that if that's okay for you!